src/xdocs/checks.xml

```diff
@@ -41,15 +41,27 @@
             </td>
           </tr>
           <tr>
-            <td><a href="config_annotation.html#AnnotationLocation">AnnotationLocation</a></td>
+            <td>
+              <a href="checks/annotation/annotationlocation.html#AnnotationLocation">
+                AnnotationLocation
+              </a>
+            </td>
             <td>Checks location of annotation on language elements.</td>
           </tr>
           <tr>
-            <td><a href="config_annotation.html#AnnotationOnSameLine">AnnotationOnSameLine</a></td>
+            <td>
+              <a href="checks/annotation/annotationonsameline.html#AnnotationOnSameLine">
+                AnnotationOnSameLine
+              </a>
+            </td>
             <td>Checks that annotations are located on the same line with their targets.</td>
           </tr>
           <tr>
-            <td><a href="config_annotation.html#AnnotationUseStyle">AnnotationUseStyle</a></td>
+            <td>
+              <a href="checks/annotation/annotationusestyle.html#AnnotationUseStyle">
```

Comment:
Does `run-link-check-plugin` check if links like this are working?

https://github.com/checkstyle/checkstyle/blob/master/.ci/run-link-check-plugin.sh

Reply:
Is there some easy way to run this locally? I would just test it out.

Reply:
it has very long list of suppression, it catching some, it does not run PR as it is very long in execution , fragile on execution and does not catch any functional issues.
We can restore it for summer in PR CI or make just another "Github, check links in website"/"Github, run <name of action>" (preferable). 

So it was moved to master branch only, to fix problems eventually.
that is why I created https://github.com/orgs/checkstyle/projects/5/views/2?pane=issue&itemId=29406306

Reply:
it might make sense to remove all excludes of any kind and run it ones to see current state.